Rubberduck.Parsing/Symbols/Declaration.cs

```diff
@@ -460,7 +460,7 @@ public bool AsTypeIsBaseType
         {
             get
             {
-                return string.IsNullOrWhiteSpace(AsTypeName) || BASE_TYPES.Contains(_asTypeName.ToUpperInvariant());
```

Comment:
The `BaseType` array is different. Well it should have been exposed as an `IEnumerable<string>`, but the point is that these values *have* to be `static`, and they gotta be *somewhere*. `Declaration` seems just convenient... for now. Eventually we'll need to implement a proper *ambient context*.